tests/_os_update.pm

```diff
@@ -6,12 +6,19 @@ use utils;
 sub run {
     my $self = shift;
 
+    $self->root_console();
+
+    # If we‘re wanting to update to master, we need to switch from production
+    # to dev.
+    my $update_to_stage = get_var('OS_UPDATE_TO_STAGE');
+    my $update_to_stage_password = get_var('OS_UPDATE_TO_STAGE_PASSWORD');
+    script_run("eos-stage-ostree $update_to_stage $update_to_stage_password", 180);
```

Comment:
I'm not exactly sure how this will work when the password is empty. Since you're always passing `OS_UPDATE_TO_STAGE_PASSWORD` in the request, `get_var` will return an empty string. I guess since you're just building a big string in perl it will just have a trailing space in that case. Correct?

Reply:
Correct.